concepts/05 UI Components/TabPanel/01 Getting Started with TabPanel/40 Navigate Between Tabs.md

```diff
@@ -0,0 +1,175 @@
+Specify the following properties to configure navigation between tabs:
+
+- [swipeEnabled](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/#swipeEnabled)      
+Specifies whether users can switch between views with the swipe gesture.
+
+- [loop](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/#loop)      
+Specifies whether to loop views.
+
+- [animationEnabled](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/#animationEnabled)      
+Specifies whether to animate the change of the current view.
+
+- [selectedIndex](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/#selectedIndex)     
+Specifies the index of the currently selected tab. Use this property to switch between tabs programmatically. In this tutorial, this property's value depends on the the RadioGroup's selected item (see `tabSwitcherRadioGroup`). 
+
+- [onSelectionChanged](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/#onSelectionChanged)        
+Allows you to specify a function that is executed when the selected tab changes. You can access the selected tab's data within this function. In the code below, this function binds the TabPanel's **selectedItem** with RadioGroup's [value](/Documentation/ApiReference/UI_Components/dxRadioGroup/Configuration/#value) properties for frameworks that do not support two-way binding.
```

Comment:
We're going to split the site be frameworks soon, and this description will give us a headache. Seeing that onSelectionChanged is only needed for this particular use-case, I think we can remove this description.

```suggestion
```